app/services/pull_request.rb

```diff
@@ -34,4 +34,8 @@ def on_hold_label?(label)
     label.downcase == ON_HOLD.downcase
   end
 
+  def blacklisted?
+    User.where(github_name: username, blacklisted: true).present?
```

Comment:
I think you can use https://apidock.com/rails/ActiveRecord/Base/exists%3F/class, not sure if the current solution brings the users to memory but I'm sure `exists` doesn't and it's more straightforward.

Reply:
Yes, that's better.